src/ChatHeader.js

```diff
@@ -0,0 +1,25 @@
+import { Avatar } from '@material-ui/core'
+import React from 'react'
+import SearchIcon from '@material-ui/icons/Search'
+import AttachFileIcon from '@material-ui/icons/AttachFile'
+import MoreVertIcon from '@material-ui/icons/MoreVert'
+import './ChatHeader.css'
+
+export function ChatHeader() {
+    return (
+        <div className='chatHeader__container'>
+            <div id='chat__info'>
+                <Avatar className='avatar__icon'/>
```

Comment:
Avatar should not come in chat__info
As, chat info suggest the information of chat and we use this class only to style the info and not avatar